packages/cli/src/CLI.ts

```diff
@@ -1,11 +1,16 @@
 import type { PrismaConfigInternal } from '@prisma/config'
-import { ensureBinariesExist } from '@prisma/engines'
+import { Debug } from '@prisma/debug'
+import { ensureNeededBinariesExist } from '@prisma/engines'
+import { download } from '@prisma/fetch-engine'
```

Comment:
`download` is passed explicitly to `ensureNeededBinariesExist` to perform safe assertions in `packages/engines/ensureNeededBinariesExist.test.ts`